src/elements/button/button.ts

```diff
@@ -0,0 +1,76 @@
+///<reference path="../../../typings/angularjs/angular.d.ts"/>
+///<reference path="../../../typings/angularjs/angular-animate.d.ts"/>
+
+'use strict';
+
+class SmButtonController {
+  static $inject = ['$animate'];
+
+  constructor(public $animate: ng.IAnimateCallbackObject) { }
```

Comment:
yes, I was thinking to create tsconfig.json, good point. I created the build that only has only semantic-ui-angular distribution in the output, so in order to actually preview you need to setup a separate demo project and bring angular, angular-animate and angular-aria as these are main dependencies. I will update README with installations instructions once we setup everything. Also, I was thinking in creating a docs page using github pages, I think I can work on it as next task, so that we can have the demo preview page. What do you think ?